### PR TITLE
BFD-3373: Updates BlueButton Static Site with new RDA fields

### PR DESCRIPTION
**Pull request by meliGuzman**
**JIRA Ticket:**
[BFD-3373](https://jira.cms.gov/browse/BFD-3373)

**User Story or Bug Summary:**

Update BlueButton Static Site with new RDA fields.

### What Does This PR Do?

Replaces the file _data/rda_api_data_dictionary.csv in BB static site repo with the exported CSV file of tab B) Data Dictionary from the RDA API Data dictionary 

### What Should Reviewers Watch For?

If you're reviewing this PR, please check these things, in particular:

Test the static site locally to verify new variables can be loaded in browser:
- NDC
- NDC-QTY
- NDC-QTY-QUAL
- DTL-NDC
- NDC-UNIT-COUNT

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence: N/A.
    * Does this PR add any new software dependencies? *No**.
    * Does this PR modify or invalidate any of our security controls? **No**.
    * Does this PR store or transmit data that was not stored or transmitted before? **No**.
* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR should not be merged unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons? **No**.

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following pre-requisite changes have been fully deployed:

### Submitter Checklist

I have gone through and verified that...:

* [X] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [X] I have named this PR and its branch such that they'll be automatically be linked to the (most) relevant Jira issue, per: <https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html>.
* [X] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [X] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [X] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [X] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. BFD, SLS, etc.).
* [X] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

**Comment by jimmyfagan** [comment]
Hi Melissa, still pretty new to the static site, to help understand this review a bit more, can you point us to where these updates will be visible in the site?

**Comment by meliGuzman** [comment]
hey @jimmyfagan ! Yeah of course so when you have docker running navigate to http://localhost:4000/resources/variables/x/ where x is the name of one of the fields that i listed in the PR

**Comment by jimmyfagan** [comment]
Same as Logan, even after doing a jekyll build, I wasn't able to find the pages under `http://localhost:4000/resources/variables/`. I think this is our first time running the static site locally, so we might be missing something, can you confirm the specific name of say the NDC-QTY page, and maybe a screenshot of what we should be seeing there? If we still can't find it, we might just need to do a huddle to walk through starting this up locally.

**Comment by meliGuzman** [comment]
Hey @jimmyfagan and @loganbertram, I gave the wrong link! Sorry about that! You'll find them here:

http://localhost:4000/resources/variables/mcs/NDC_UNIT_COUNT/
http://localhost:4000/resources/variables/mcs/DTL_NDC/
http://localhost:4000/resources/variables/fiss/ndc_qty/
http://localhost:4000/resources/variables/fiss/ndc_qty_qual/
http://localhost:4000/resources/variables/fiss/ndc/
http://localhost:4000/resources/variables/fiss/ADM_TYPE_CD/

Some variables are separated by upstream source! Also, it's case insensitive btw!

**Comment by loganbertram** [comment]
@meliGuzman No worries! Those all work as expected on my machine. I'll go ahead and approve because this all looks good.

**Review by loganbertram — APPROVED** [review]

File: _data/rda_api_data_dictionary.csv [hunk 1]
```diff
@@ -23,7 +23,7 @@ FISS,Adjustment Medicare Beneficiary Identifier,ADJ-MBI,fiss_claims.adj_mbi,Bene
 FISS,Adjustment Medicare Beneficiary Identifier (MBI) Indicator,ADJ-MBI-IND,fiss_claims.adj_mbi_ind,Beneficiary information,n/a,This field identifies the adjustment/cancel claim that was submitted with a Health Insurance Claim (HIC) Number or MBI number.,,,Y,"Alphanumeric, 1 character
 H = The HIC was submitted on the adjustment/cancel claim
 M = The MBI was submitted on the adjustment/cancel claim
-Blank = This field will be blank on adjustment/cancel claims created prior to 7/01/18 and non-adjustment/cancel claims.",FSRCCLXB,0.5,,,,,
+Blank = This field will be blank on adjustment/cancel claims created prior to 7/01/18 and non-adjustment/cancel claims.",FSRCCLXB,0.5,,,,,Does this field get populated on the original claim or the child claim that is adjusting/canceling the parent claim?
```
Comment (jimmyfagan) [inline _data/rda_api_data_dictionary.csv:26]: Was this meant to be part of this changeset? I couldn't find this on any of the pages when running this locally, but it seemed like maybe this was meant to be a note-to-self and not something that would be published.

**Review by jimmyfagan — COMMENTED** [review]
Thanks for the updated links! Just had one question that you might want to look at before merging, but otherwise, this looks good. I suppose we'll need to plan a static site deployment to get this rolled out, if you have any feedback around the urgency/priority of that, let me know!

File: _data/rda_api_data_dictionary.csv (hunk 1 above)
Comment (meliGuzman) [reply _data/rda_api_data_dictionary.csv:26]: @jimmyfagan This is a a lil weird change but it's from the RDA team. When we get new RDA fields we have to replace the file _data/rda_api_data_dictionary.csv in the BB static site repo with the exported CSV file from the tab B) Data Dictionary from RDA's data dictionary [RDA API Data dictionary](https://docs.google.com/spreadsheets/d/19CGXLO9SGTgaSvACRVuuOatvs4z1NXHzxtI-zt_j8Qc/edit#gid=283461576) I believe it should be fine

**Comment by meliGuzman** [comment]
@jimmyfagan Thanks for pointing out the deployment timeline! So yes, this will have to go out within the same sprint as our ticket BFD-3373 which will likely be at the end of this sprint. Is that possible for you guys?

File: _data/rda_api_data_dictionary.csv (hunk 1 above)
Comment (jimmyfagan) [reply _data/rda_api_data_dictionary.csv:26]: Ah okay, that makes sense, thank you!

**Review by jimmyfagan — APPROVED** [review]
Looks good! Just keep us updated on the progress of that other ticket so we can plan the deployment, thanks!

**Pull request merged**